docs/government.md

```diff
@@ -240,6 +240,10 @@ pageClass: routes
 
 </Route>
 
+### 科学技术政策办公室
+
+<Route author="LyleLee" example="/whitehouse/ostp" path="/whitehouse/ostp"/>
+
```

Comment:
Please also update `docs/en/government.md`

Reply:
Thanks to point out. Please review.